client/plots/hierCluster.js

```diff
@@ -787,7 +787,8 @@ export async function getPlotConfig(opts = {}, app) {
 				tvs: {
 					values: [{ dt: 3 }]
 				}
-			}
+			},
+			type: 'hierCluster'
```

Comment:
@congyu-lu would recovered gene expression plots have this `group.type`? May need to add code to assign this group.type for recovered sesions/overrides where a group's name matches the recovered config.termGroupName.